src/Constants.watch.cs.in

```diff
@@ -63,6 +63,6 @@ namespace ObjCRuntime {
 
 		// WatchOS 6
 		public const string SoundAnalysisLibrary = "/System/Library/Frameworks/SoundAnalysis.framework/SoundAnalysis";
-
+		public const string LinkPresentationLibrary = "/System/Library/Frameworks/LinkPresentation.framework/LinkPresentation";
```

Comment:
Headers are **not** available on watchOS (only iOS and macOS)
Sadly you can't trust availability macros over header presence :|

Reply:
you did not add it to `src/frameworks.sources` do it's one less thing to remove ;-)